lib/http.js

```diff
@@ -202,8 +202,10 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       }
       const {profileAgentId} = req.params;
       const {profileAgent} = await profileAgents.get({id: profileAgentId});
-      const {capabilitySet} = await capabilitySets.get({profileAgentId});
-      const {zcaps: capabilities} = capabilitySet;
+
+      // all capabilities in the `zcaps` map will be delegated
+      const capabilities = Object.values(profileAgent.zcaps);
```

Comment:
If this is the endpoint I think it is, then what we want to do is delegate a single zcap for using the profile agent's zcap key (capability invocation key).